testsuite/tests/libratest/smoke_test.rs

```diff
@@ -1141,15 +1141,6 @@ fn test_e2e_modify_publishing_option() {
         .disable_custom_script(&["disable_custom_script"], true)
         .unwrap();
 
-    // mint another 10 coins after restart
```

Comment:
This is a pretty unfortunate consequence of this PR: If a network tries to flip from custom script execution to closed allowlist, no scripts could be executed at all since there were nothing in the allowlist.